pilaf/storybook/stories/index.ts

```diff
@@ -0,0 +1,2 @@
+import "./Button/Button.stories";
+// import './Welcome/Welcome.stories';
```

Comment:
Is this one supposed to be commented out?

Reply:
Yeah, for some reason `welcome.stories` does seem to cause trouble so I commented it out just so it wouldn't be displayed in the navigation (this persisted on another fresh rn app).

Reply:
Alright. Does it give any errors or anything?

Reply:
The given component by storybook is class based, I guess TS doesn't like how it's written (The errors are now shown in JS).
![image](https://user-images.githubusercontent.com/34408108/111893047-a2edfb00-89c5-11eb-9c29-3468a3dda6a4.png)

Reply:
Welcome.stories is just a demo stories. You can remove all generated js stories and add one tsx stories as example.

![image](https://user-images.githubusercontent.com/8739777/111895834-89749f80-8a15-11eb-977b-905555a2737c.png)

Reply:
Got ya.